cime_config/testdefs/testlist_clm.xml

```diff
@@ -1672,6 +1672,15 @@
       <option name="tput_tolerance">0.5</option>
     </options>
   </test>
+  <test name="SMS_D_Vnuopc" grid="CLM_USRDAT" compset="I1PtClm50Bgc" testmods="clm/NEON_YELL">
+    <machines>
+      <machine name="cheyenne" compiler="intel" category="aux_clm"/>
+    </machines>
+    <options>
+      <option name="wallclock">00:20:00</option>
+      <option name="tput_tolerance">0.5</option>
+    </options>
+  </test>
```

Comment:
It turns out YELL data isn't available yet. But, we have at least a year of NIWO, so I'll change the test to NIWO and if needed only run for that first year.